Makefile.dep

```diff
@@ -325,3 +325,8 @@ ifneq (,$(filter cpp11-compat,$(USEMODULE)))
   USEMODULE += timex
   FEATURES_REQUIRED += cpp
 endif
+
+ifneq (,$(filter hih6130,$(USEMODULE)))
+  USEMODULE += vtimer
+  FEATURES_REQUIRED += periph_i2c
```

Comment:
Honestly I don't know. I think most sensors define `FEATURES_REQUIRED` just in the makefile of their application. But doesn't mean that is the best solution. Are there any opinions by others?